src/org/opendatakit/briefcase/util/FileSystemUtils.java

```diff
@@ -74,11 +78,46 @@
   static final String ENCRYPTED_FILE_EXTENSION = ".enc";
   static final String MISSING_FILE_EXTENSION = ".missing";
 
+  static final File cacheFile = new File(getFormsFolder().getParent(), "cache.ser");
```

Comment:
This line could be made a little clearer with the following amendment.
`static final File cacheFile = new File(getBriefcaseFolder(), "cache.ser");`